src/applications/gi-sandbox/containers/Modals.jsx

```diff
@@ -304,6 +304,34 @@ export class Modals extends React.Component {
         </div>
       </Modal>
 
+      <Modal
```

Comment:
All modal content now exists in https://github.com/department-of-veterans-affairs/vets-website/tree/master/src/applications/gi-sandbox/components/content/modals